docs/api/catalog-resource.md

```diff
@@ -244,6 +245,15 @@ time before the catalog item's commit timestamp.
 The `packageHashAlgorithm` is a string defined by the server implementation representing the hashing algorithm used to
 produce the `packageHash`. nuget.org always used the `packageHashAlgorithm` value of `SHA512`.
 
+The `packageTypes` property will only be present if a package type was specified by the author. If it is present, it will always have at least one (1) entry. Each item in the `packageTypes` array is a JSON object with the following properties:
```

Comment:
Should we update the sample catalog leaf response to include the `packageTypes` property?